src/main/java/ua/danit/final_project/entities/Employee.java

```diff
@@ -53,4 +53,8 @@
 
   @Column(name = "e_info")
   private String info;
+
+  public boolean hasUser() {
+    return !(user == null);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the opposite operator ("!=") instead. ([squid:S1940](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1940))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=133&issues=AWY3Io01M63is4RDJ_0X&open=AWY3Io01M63is4RDJ_0X)

[ISSUE_KEY]: AWY3Io01M63is4RDJ_0X